go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -237,11 +244,19 @@ func (c *Controller) handleErr(err error, key interface{}) {
 // That is because such work will be performed in syncService, so all that is needed here is the ability
 // to distinguish what is present in ovn database and this 'dirty' initial value.
 func (c *Controller) initTopLevelCache() error {
+	var err error
+
 	c.alreadyAppliedLock.Lock()
 	defer c.alreadyAppliedLock.Unlock()
 
-	// first, list all load balancers and their respective services
-	services, lbs, err := getServiceLBs(c.nbClient)
+	// First list all the templates.
+	c.alreadyAppliedTemplates, err = libovsdbops.ListTemplates(c.nbClient)
```

Comment:
what's the purpose of this (`c.alreadyAppliedTemplates`) cache? can't we just pass the `ListTemplates` output directly into `getServiceLBs`? not sure I follow the purpose of holding this in mem on the controller struct. I don't see this being used anywhere...

Reply:
You're right.  This was related to the original version of the PR, back when there were multiple caches of LBs.  Removed.